README.md

```diff
@@ -14,12 +18,20 @@ All available templates (`expr` means the expression on which the template is ap
 | **.else**         | `if !expr` |
 | **.nil**         | `if expr == nil` |
 | **.notnil**      | `if expr != nil` |
+| **.nilr**      | `if expr == nil  { return expr }` |
```

Comment:
It seems value returned will always be nil. How about returning `\${0}` instead?

Reply:
We need to add some explanation to this. I considered `nilr` as this.

`expr.nilr ==>`

```
if myVal == nil {
    return myVal
}
``` 
if the value of object is nil then return the object which is also nil. In case of returning the `\${0}` I don't see any use for it as there is no value in it in this case.

Correct me if I am wrong about `\${0}`.